tapis-project/tapis-ui
# task/TUI-61,62,69 - move file listing and launcher into section details

## Overview: ##
WIP
This moves the FileListing and Launcher into Section details for SystemList and the AppListing. Also Added Icons in...

## Related Github Issues: ##

* [TUI-61](https://github.com/tapis-project/tapis-ui/issues/61)

## Testing Steps: ##
1. npm ci
2. go to Systems
3. go to Apps
4. check out icons
